lib/aptible/cli/helpers/tunnel.rb

```diff
@@ -1,5 +1,6 @@
 require 'socket'
 require 'open3'
+require 'win32/process' if RUBY_PLATFORM =~ /mswin|mingw/i
```

Comment:
Regardless of whether we choose to use Gem's implementation here, can we avoid using the same regexp in multiple places? If we ever need to change the pattern for some reason, it would be preferable not to have to change it in multiple places (we might forget to do so, and our tests will miss it unless they happen to not match the old pattern and match only the new one, which seems unlikely).